estate/models/estate_property.py

```diff
@@ -0,0 +1,40 @@
+from odoo import fields, models
+
+
+class Estate(models.Model):
+    _name = "estate.property"
+    _description = "Properties of estate entities."
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    active = fields.Boolean(default=True)
+    postcode = fields.Char()
+    date_availability = fields.Date(
+            copy=False,
+            default=fields.Date.add(fields.Date.today(), fields.Date.today(), months=3))
```

Comment:
```suggestion
            default=fields.Date.add(fields.Date.today(), months=3))
```
I'm surprised it even works, not sure why this second parameter is there for :rofl: